packages/social/config/default.json

```diff
@@ -5,6 +5,7 @@
     "loginDisabled": false,
     "logo": "./logo.svg",
     "apiServer": "https://${VITE_SERVER_HOST}:${VITE_SERVER_PORT}",
+    "corsServer": "https://${VITE_SERVER_HOST}:${VITE_CORS_SERVER_PORT}",
```

Comment:
I don't believe this is needed, since the `social` package is our ARC-related code that's getting pull out of the main repo soon and moved to its own package.